streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceiverTracker.scala

```diff
@@ -119,9 +118,17 @@ class ReceiverTracker(ssc: StreamingContext, skipReceiverLaunch: Boolean = false
     }
   }
 
-    /** Clean up metadata older than the given threshold time */
-  def cleanupOldMetadata(cleanupThreshTime: Time) {
+  /** Clean up the data and metadata of old blocks and batches */
```

Comment:
It would be good if this explained what happens for blocks which are assigned _exactly_ to the thresholdTime. The old doc seemed to imply that these would be spared (i.e. the block needs to be strictly older). It would be good to clarify this in the new one also. Actually throughout this entire call chain related to cleaning old things, it would be good to clarify in the docs what the boundary behavior is.